docs/server/source/appendices/commands.rst

```diff
@@ -0,0 +1,18 @@
+######################
```

Comment:
Do we really need this since we alredy have https://github.com/bigchaindb/bigchaindb/blob/v0.9.4/docs/server/source/server-reference/bigchaindb-cli.md

This reminds me that we should also document the logging cmd option if we didn't already.